README.md

```diff
@@ -1,2 +1,76 @@
 # skeleton
 A javascript skeleton project utilizing industry popular tooling.
+
+## Installation
+
+### Installing Required Packages
```

Comment:
Can you split this between 2 sections:
1) Environment Setup
2) Project Installation

To distinguish between (1) being a set up once for the environment and (2) being for each individual project